.github/workflows/ci.yml

```diff
@@ -15,9 +16,14 @@ jobs:
           - '2.7'
           - '3.0'
           - 'head'
+        gemfile:
+          - gemfiles/rspec_rails_4_0.gemfile
+          - gemfiles/rspec_rails_5_0.gemfile
```

Comment:
Should we also add the check for `rspec_rails_6_0` which is the current stable version ?

Reply:
I forgot D: